lib/src/frame.dart

```diff
@@ -30,6 +30,12 @@ final _v8UrlLocation = new RegExp(r'^(.*):(\d+):(\d+)|native$');
 final _v8EvalLocation =
     new RegExp(r'^eval at (?:\S.*?) \((.*)\)(?:, .*?:\d+:\d+)?$');
 
+// foo._bar@http://pub.dartlang.org/stuff.js:18056:12
```

Comment:
Let's remove this and the third line on this comment (since those two patterns are not matched by the regexp here).

Reply:
ok, removed